jobs/build/release/Jenkinsfile

```diff
@@ -99,6 +99,12 @@ node {
             String errata_url
             Map release_obj
 
+            currentBuild.displayName += "- ${name}"
+            if (params.DRY_RUN) {
+                currentBuild.displayName += " (dry-run)"
```

Comment:
How about prepending `[DRY RUN] ` to the description? It will be consistent with the `release` job.

Reply:
done: https://github.com/openshift/aos-cd-jobs/pull/2037/commits/3f9557c608fed8bb0edd56f162fc7630125b76ef#diff-cbfda85ed8407cced5ac8810a72001f0R105